### PR TITLE
2024 10 04 jk qene mamer

### PR DESCRIPTION
**Pull request by karljonaskarlsson**
This is clearly some kind of poetic composition, but it appears not to be rhymed, so I'm not sure if it can be classified as a qene in the strict sense. It's from the same manuscript as the rest of the Uppsala qene, with images here: https://www.alvin-portal.org/alvin/imageViewer.jsf?dsId=ATTACHMENT-0310&pid=alvin-record:280997

### CODE REVIEW

**Review by nafisa-valieva — APPROVED** [review]
very interesting! were they impessed by him really or they are flirting? :-)

File: new/LIT7092QeneMamer.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7092QeneMamer" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ማእምር፡ ወለባዊ፡ ብዕሴ፡ እ<hi rend="ligature">ግዚ</hi>አብሔር፤</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Māʾmǝr wa-labbāwi bǝʿse ʾƎgziʾabǝḥer</title>
```
Comment (eu-genia) [inline new/LIT7092QeneMamer.xml:9]: No mark up in title

File: new/LIT7092QeneMamer.xml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>Eulogistic unrhymed composition in nine lines. According to the manuscript,
+                    it was composed by <persName ref="PRS14575KidanaMaryam"/> in the honour 
+                    of <persName ref="PRS14576AndersSvensson"/> (called <foreign xml:lang="gez">ቄስ፡ ሜንሶን፡</foreign>).</p>
```
Comment (eu-genia) [inline new/LIT7092QeneMamer.xml:39]: in honour of (delete the)

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT7092QeneMamer.xml (hunk 1 above)
Comment (karljonaskarlsson) [reply new/LIT7092QeneMamer.xml:9]: Sorry, thank you!

**Review by eu-genia — APPROVED** [review]

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Pull request merged**